bin/ghe-restore

```diff
@@ -343,10 +343,14 @@ fi
 
 # Clean up all stale replicas
 if ! $CLUSTER; then
-  inactive_nodes=$(echo "ghe-spokes server show inactive --json | jq -r '.[] | select(.host | contains(\"git-server\")).host' | sed 's/^git-server-//g'" | ghe-ssh "$GHE_HOSTNAME" -- /bin/bash)
-  if [ -n "$inactive_nodes" ]; then
+  restored_uuid=$(cat $GHE_RESTORE_SNAPSHOT_PATH/uuid)
+  other_nodes=$(echo "ghe-spokes server show --json | jq -r '.[] | select(.host | contains(\"git-server\")).host' | sed 's/^git-server-//g'" | ghe-ssh "$GHE_HOSTNAME" -- /bin/bash)
```

Comment:
To reduce lines of code by 3, would we just add a `| grep -v $restored_uuid` to the end of the large collection of piped commands here? Or would this reduce the readability too much?